src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -7,10 +7,11 @@ import {
   INCLUDE_UNSUPPORTED,
 } from '#lib/logger.const';
 import nodeLogger from '#lib/logger.node';
+import { addOverrideQuery } from '../../../../../utils/overrideRendererOnTest';
```

Comment:
could I suggest we add a dir alias for this? perhaps `#routeUtils` could be done in a follow up PR if you like.

Reply:
Yep I agree. I can create a follow up issue for it 👍